Web.md

```diff
@@ -60,15 +60,18 @@ as described in the [`WebAssembly.Module` constructor](#webassemblymodule-constr
 On success, the `Promise` is [fulfilled](http://tc39.github.io/ecma262/#sec-fulfillpromise)
 with the resulting `WebAssembly.Module` object. On failure, the `Promise` is
 [rejected](http://tc39.github.io/ecma262/#sec-rejectpromise) with a
-`WebAssembly.CompileError`.
+`WebAssembly.CompileError` or `TypeError`, depending on the type of failure.
```

Comment:
I went with what we had for instantiate, but happy to change both in this manner if others agree. Eventually we need to redo both these descriptions in a more formal matter once you all figure out how you want to define your objects (see #1048).